docs/orchestrating-elastic-stack-applications/elasticsearch/volume-claim-templates.asciidoc

```diff
@@ -31,7 +31,27 @@ spec:
         storageClassName: standard
 ----
 
-ECK automatically deletes PersistentVolumeClaim resources if they are not required for any Elasticsearch node. The corresponding PersistentVolume may be preserved, depending on the configured link:https://kubernetes.io/docs/concepts/storage/storage-classes/#reclaim-policy[storage class reclaim policy].
+== Controlling volume claim deletion
```

Comment:
```suggestion
== Controlling persistent volume claim deletion
```

Reply:
I think I leave it as is as we are consistently just speaking of "volume claims" in the titles of this chapter. Persistent is implied.